scss/stanford_landing_page.scss

```diff
@@ -0,0 +1,148 @@
+// Default styles for Landing Page content type
+.content-body .field.field-name-field-s-lp-item-link,
+.field.field-name-tile-display-image p {
+  margin-bottom: 0;
+}
+
+.field-name-field-s-lp-2nd-links .field-item {
+  margin-bottom: 1em;
+}
+
+// Responsive grids for blocks and tiles
+.node-stanford-landing-page.large-landscape .field-name-field-landing-page-item > .field-items > .field-item,
+.node-stanford-landing-page.stanford-4-col-header .field-name-field-landing-page-item > .field-items > .field-item {
+  display: inline-block;
+  margin-right: 3%;
+  width: 31.333%;
+  max-width: 370px;
+  margin-bottom: 1.5em;
+  vertical-align: top;
+}
+
+.node-stanford-landing-page.large-landscape .field-name-field-landing-page-item > .field-items > .field-item:nth-child(3),
```

Comment:
Please use the 3n convention for this one instead of 3, 6, 12, 15.

see: http://www.w3schools.com/cssref/tryit.asp?filename=trycss3_nth-child_formula